pkgs/by-name/ge/geph/package.nix

```diff
@@ -0,0 +1,114 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  libxkbcommon,
+  openssl,
+  rust-jemalloc-sys-unprefixed,
+  sqlite,
+  vulkan-loader,
+  wayland,
+  iproute2,
+  iptables,
+  libglvnd,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+let
+  binPath = lib.makeBinPath [
+    iproute2
+    iptables
+  ];
+in
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "geph5";
+  version = "0.2.61";
+
+  src = fetchFromGitHub {
+    owner = "geph-official";
+    repo = "geph5";
+    rev = "geph5-client-v${finalAttrs.version}";
+    hash = "sha256-qy1E5x5Fn+xwS5st6HkMrJu9nksXQQIyJf97FvNOKO4=";
+  };
+
+  cargoHash = "sha256-r97DsSsqp/KtgqtYQe92nz2qaOBcJF6w9ckfxpk8Cxg=";
+
+  patches = [ ./test-fix.patch ];
+
+  postPatch = ''
+    substituteInPlace binaries/geph5-client/src/vpn/*.sh \
+      --replace-fail 'PATH=' 'PATH=${binPath}:'
+  '';
+
+  nativeBuildInputs = [
+    pkg-config
+    copyDesktopItems
+  ];
+
+  buildInputs = [
+    openssl
+    rust-jemalloc-sys-unprefixed
+    sqlite
```

Comment:
I think you need to add `env.LIBSQLITE3_SYS_USE_PKG_CONFIG = "1";`, otherwise this has no effect.